app/views/welcome/merchant_dashboard.html.erb

```diff
@@ -1 +1 @@
-<h1>Merchant Dashboard</h1>
+<%= render 'shared/merchant_dashboard' %>
```

Comment:
This looks like a partial, I am curious as to why make it a partial vs rendering it in entirety in this file. Is that partial used elsewhere?